angular15-microfrontends-lazy-components/e2e/checkAngular15MicroLazyComponentsApps.cy.ts

```diff
@@ -0,0 +1,542 @@
+import { blocks, fields, buttons, baseSelectors, alertMessages } from './../../cypress/common/selectors';
+import { Constants } from '../../cypress/fixtures/constants';
+import { BaseMethods } from '../../cypress/common/base';
+
+const basePage: BaseMethods = new BaseMethods()
+
+const appsData = [
+    {
+        appNameText: Constants.elementsText.mdmfShell.name,
+        headerText: Constants.elementsText.mdfmShellHeader,
+        isWelcomeText: true,
+        welcomeText: Constants.commonPhrases.mdmfShellWelcome.replace('11', '15'),
+        paragraphText: Constants.commonPhrases.mdmfShellParagraph,
+        tableHeaderText: Constants.elementsText.mdmfAppTableHeader,
+        isCardBody: false,
+        sharedHeader: '',
```

Comment:
not quite sure about it, what is the point of empty field check?

Reply:
It's only for the if statement, cause my apps is different a little

Reply:
Fixed